src/pages/docs/connect/states.mdx

```diff
@@ -409,3 +409,19 @@ While an SDK will not automatically attempt to reconnect in the `FAILED` state,
 Other classes of error are nonfatal. For example, a client may have network connectivity issues. An SDK will attempt to automatically reconnect and recover from these sort of issues, as detailed in the `DISCONNECTED` and `SUSPENDED` explanations in the [Available connection states](#connection-states) section.
 
 If message continuity is lost in the process, e.g. because you have been disconnected from Ably for more than two minutes, the SDK will notify you through the [`resumed`](#resume) flag mechanism.
+
+<If lang="swift,objc">
```

Comment:
I'd be tempted to say this shouldn't be hidden based on language. It would be nice to make it mobile generic in future too.